Add option for volumes and volumes_from

KKoukiou opened the pull request.

KKoukiou committed:
Add option for volumes and volumes_from

chrismeyersfsu commented:
Ideally, tests would be written for both interfaces, the `inventory` style interface and the `cloud` style interface. You have included a very good `cloud` style interface test. Mind including the `inventory` style interface tests?

KKoukiou commented:
I added tests for both style interfaces, checked both new added parameters (volumes & volumes_from) and also that there is no problem if volume related parameters are omitted.
Also I added some info in the README.md file for the new parameters.

chrismeyersfsu commented:
Awesome !!!
